model/sparql/GenericSparql.php

```diff
@@ -429,6 +430,10 @@ private function generateConceptInfoQuery($uris, $arrayClass, $vocabs) {
   UNION
   {
    ?uri ?p ?o .
+   OPTIONAL {
+     BIND(datatype(?o) AS ?dt)
+     ?dt rdfs:label ?dtlabel
```

Comment:
I think this block should check that there is a datatype. Otherwise `?dt` remains unbound and then the query will pick up all `rdfs:label` triples, which wasn't intended.

So add this clause inside the OPTIONAL block:

    FILTER(bound(?t))

Reply:
fixed